client/blocks/payment-request/payment-request-express.js

```diff
@@ -120,15 +121,31 @@ const PaymentRequestExpressComponent = ( {
 	}
 
 	return (
-		<PaymentRequestButtonElement
-			onClick={ onPaymentRequestButtonClick }
-			options={ {
-				// @ts-ignore
-				style: paymentRequestButtonStyle,
-				// @ts-ignore
-				paymentRequest,
-			} }
-		/>
+		// We wrap using a div because we can't pass a 'style' prop to PaymentRequestButtonElement.
+		// The pointerEvents hack here is an attempt to improve the UX while we're sending an API
+		// request for the cart. Instead of the button not being clickable and showing a mouse pointer
+		// that indicates that the button is clickable (the hand), instead we just show the regular
+		// mouse pointer.
+		// We'd prefer to just disable the ExpressPaymentButton through the Blocks API, but that's not
+		// possible at the moment.
+		// - @reykjalin
+		<div
+			style={
+				isUpdatingPaymentRequest
+					? {
+							pointerEvents: 'none',
+					  }
+					: {}
+			}
+		>
+			<PaymentRequestButtonElement
+				onClick={ onPaymentRequestButtonClick }
+				options={ {
+					style: paymentRequestButtonStyle,
+					paymentRequest,
+				} }
+			/>
+		</div>
```

Comment:
How about doing something like this? (very very hacky I know 😞)...

The benefit of this approach is that we keep the button "disabled" until is clickable... I know is just 1 second and maybe not worth it, so feel free to disregard this suggestion

```suggestion
		<div
			{ ...( isUpdatingPaymentRequest && {
				className: 'wc-block-components-loading-mask',
			} ) }
		>
			<div
				{ ...( isUpdatingPaymentRequest && {
					className: 'wc-block-components-loading-mask__children',
				} ) }
			>
				<PaymentRequestButtonElement
					onClick={ onPaymentRequestButtonClick }
					options={ {
						style: paymentRequestButtonStyle,
						paymentRequest,
					} }
				/>
			</div>
		</div>
```

Reply:
Works like a charm, although I added the child element class directly onto the Stripe Element itself 🙂 

Fixed in afa9bf6b259ebaa4324d023965a8d7f092af15c1!

Reply:
Nice suggestion by the way @diegocurbelo ! I couldn't figure out the necessary class combinations for this 👏